main.go

```diff
@@ -242,6 +250,14 @@ func ParseCmdLine() *State {
 			}
 		}
 
+		// prompt for password if needed
+		if valid && s.Username != "" && s.Password == "" {
+			print("Auth Password: ")
+			reader := bufio.NewReader(os.Stdin)
+			s.Password, _ = reader.ReadString('\n')
+			s.Password = s.Password[:len(s.Password)-1]
```

Comment:
Did you test this out on Windows? I'm tempted to use [this solution](http://stackoverflow.com/a/32768479/611) instead. Thoughts?